### DISCUSSION TITLE
Add an api that uses prebuilt slint library

### DISCUSSION THREAD

**Discussion by sigmaSd** [slint-ui/slint#1969] [General] [answered]
I enjoy how easy it is to use python with gtk or qt, no compiling needed

I tried the node api, but it seems like node install actually calls cargo to compile slint library

It would be great if it dynamically linked instead, as  I would expect no compiling from scripting languages

Also it would be great if you add a deno api

**Comment by ogoffart** [comment, marked as answer]
Hi,

You are absolutely right that we need to address these issues in order to provide the best possible developer experience.
I have created issues #1991 and #1992 in our repository to track the implementation of the npm binaries and Deno API.

However, we're not so familiar with these packages system. Is that something you would be willing to help us with?

**Reply by sigmaSd** [reply]
Thanks ! I'm mostly familiar with deno, I think I can help there, I'll subscribe to these issues, and see how are you going to start approaching binary in npm
